package.json

```diff
@@ -25,6 +25,7 @@
         "lint": "npm run lint --silent --workspaces --if-present",
         "lint:fix": "npm run lint:fix --silent --workspaces --if-present",
         "format": "prettier --write . && npm run format --silent --workspaces --if-present",
+        "update:opcua": "npx -y npm-check-updates -u --deep -f \"node-opcua*\" && npm install --workspaces",
```

Comment:
Mhh, needed/wanted ?

Reply:
Yes, in the new binding there will be a dozen of node-opcua-* modules to update, this script will become handy.

Reply:
I see  (https://github.com/eclipse/thingweb.node-wot/pull/663#pullrequestreview-874492744)

Reply:
Can't this be moved to the `binding-opcua` `package.json`?

Reply:
I'll drop this PR , I don't thnk it is needed anymore as the new binding will overwriteit.